Example/src/TagInputExample.js

```diff
@@ -5,26 +5,29 @@ import {
 } from 'react-native';
 import TagInput from 'react-native-tag-input';
 
+const inputProps = {
+  keyboardType: 'default',
+  placeholder: 'email',
+  autoFocus: true,
+};
+
 export default class TagInputExample extends Component {
   state = {
     tags: [],
+    text: "",
   };
 
   onChangeTags = (tags) => {
-    this.setState({
-      tags,
-    });
+    this.setState({ tags });
+  };
+
+  onChangeText = (text) => {
+    this.setState({ text });
```

Comment:
How about also move and tweak the "default" parseTags invocation and function to this example? like 
```
  onChangeText = (text) => {
    this.setState({ text });

    const lastTyped = text.charAt(text.length - 1);

    const parseWhen = [',', ' ', ';', '\n'];

    if (parseWhen.indexOf(lastTyped) > -1) {
      this.parseTags();
    }
  };

  parseTags = () => {
    const { text, tags } = this.state;

    const regex = /(.+)/gi;
    const results = text.match(regex);

    if (results && results.length > 0) {
      this.setState({ text: '' });
      this.onChangeTags([...new Set([...tags, ...results])]);
    }
  }
```

It may provide a friendly reference to start with :)